client/src/App/App.css

```diff
@@ -1,4 +1,5 @@
 @import '~antd/dist/antd.css';
+@import '~bootstrap/dist/css/bootstrap.min.css';
```

Comment:
lets add a high priority issue to remove bootstrap.

Reply:
its already addressed in #105

Reply:
cool.